contributing/docs-style.md

```diff
@@ -0,0 +1,343 @@
+* Link the ticket to the original contribution JIRA ticket.
+
+Looking for style guides to contribute with translating existing documentation to Chinese? Go ahead and consult [this translation specification](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications).
+
+[Back to top](#top)
+
+## Language Style
+
+
+Below, you find some basic guidelines that can help ensure readability and accessibility in your writing. For a deeper and more complete dive into language style, also refer to [The Guiding Principles of ACORN](#the-guiding-principles-of-acorn).
+
+### Addressing the User
+
+* **Use active voice.** [Active voice](https://medium.com/@DaphneWatson/technical-writing-active-vs-passive-voice-485dfaa4e498) supports brevity and makes content more engaging. If you add _by squirrels_ after the verb in a sentence and it still makes sense, you are using the passive voice.
+
+* **Use you, never we.** Using _we_ can be confusing and patronizing to some users, giving the impression that “we are all members of a secret club and _you_ didn’t get a membership invite”. Address the user as _you_.
+
+* **Avoid gender- and culture-specific language.** There is no need to identify gender in documentation: technical writing should be [gender-neutral](https://techwhirl.com/gender-neutral-technical-writing/). Also, jargon and conventions that you take for granted in your own language or culture are often different elsewhere. Humor is a staple example: a great joke in one culture can be widely misinterpreted in another.
+
+* **Avoid qualifying and prejudging actions.** For a user that is frustrated or struggling to complete an action, using words like _quick_ or _easy_ can lead to a poor documentation experience.
+
+### Using Flink-specific Terms
+Use clear definitions of terms or provide additional instructions on what something means by adding a link to a helpful resource, such as other documentation pages or the [Flink Glossary](https://ci.apache.org/projects/flink/flink-docs-master/concepts/glossary.html).
+
+[Back to top](#top)
+
+## Repository
+
+Markdown files (.md) should have a short name that summarizes the topic covered, spelled in **lowercase** and with **underscores** separating the words. The Chinese version file should have the same name as the English version, but suffixed with **.zh.md**.
+
+[Back to top](#top)
+
+## Syntax
+
+The documentation website is generated using [Jekyll](https://jekyllrb.com/docs/) and the pages are written in [Markdown](https://daringfireball.net/projects/markdown/syntax), a lightweight portable format for web publishing (but not limited to it).
+
+### Extended Syntax
+
+Markdown can also be used in combination with [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/) and plain [HTML](http://www.simplehtmlguide.com/cheatsheet.php). For example, some contributors prefer to use HTML tags for images and are free to do so with this intermix.
+
+### Front Matter
```

Comment:
The very next thing after the front matter has to be an Apache license. Every page looks like: 

```
---
title: "My Page"
nav-id: mypage
---
<!--
Licensed to the Apache Software Foundation (ASF) under one
or more contributor license agreements.  See the NOTICE file
distributed with this work for additional information
regarding copyright ownership.  The ASF licenses this file
to you under the Apache License, Version 2.0 (the
"License"); you may not use this file except in compliance
with the License.  You may obtain a copy of the License at

  http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing,
software distributed under the License is distributed on an
"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
KIND, either express or implied.  See the License for the
specific language governing permissions and limitations
under the License.
-->
```

Reply:
Regardless of which language the docs are written in (English, Chinese), the license is always English.